styles/rentry/catppuccin.user.css

```diff
@@ -0,0 +1,526 @@
+    .admonition.hint,
+    .admonition.tip {
+      color: @blue;
+      background: fade(@blue, 25%);
+    }
+    .admonition.info > .admonition-title::before,
+    .admonition.hint > .admonition-title::before,
+    .admonition.tip > .admonition-title::before {
+      @svg: escape(
+        '<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="@{blue}" viewBox="0 0 16 16"><path d="M2 6a6 6 0 1 1 10.174 4.31c-.203.196-.359.4-.453.619l-.762 1.769A.5.5 0 0 1 10.5 13h-5a.5.5 0 0 1-.46-.302l-.761-1.77a1.964 1.964 0 0 0-.453-.618A5.984 5.984 0 0 1 2 6zm3 8.5a.5.5 0 0 1 .5-.5h5a.5.5 0 0 1 0 1l-.224.447a1 1 0 0 1-.894.553H6.618a1 1 0 0 1-.894-.553L5.5 15a.5.5 0 0 1-.5-.5z"/></svg>'
+      );
+      content: url("data:image/svg+xml,@{svg}");
+    }
+    .admonition.note,
+    .admonition.important {
+      color: @green;
+      background-color: fade(@green, 25%);
+    }
+    .admonition.greentext {
+      color: @green;
+      background-color: fade(@yellow, 25%);
+    }
+    .admonition.note > .admonition-title:before,
+    .admonition.important > .admonition-title:before,
+    .admonition.greentext > .admonition-title:before {
+      @svg: escape(
+        '<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="@{green}" viewBox="0 0 16 16"><path d="M0 2a2 2 0 0 1 2-2h12a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V2zm8.93 4.588-2.29.287-.082.38.45.083c.294.07.352.176.288.469l-.738 3.468c-.194.897.105 1.319.808 1.319.545 0 1.178-.252 1.465-.598l.088-.416c-.2.176-.492.246-.686.246-.275 0-.375-.193-.304-.533L8.93 6.588zM8 5.5a1 1 0 1 0 0-2 1 1 0 0 0 0 2z"/></svg>'
+      );
+      content: url("data:image/svg+xml,@{svg}");
+    }
+    .admonition.warning {
+      color: @yellow;
+      background-color: fade(@yellow, 25%);
+    }
+    .admonition.warning > .admonition-title:before,
+    .admonition.caution > .admonition-title:before,
+    .admonition.attention > .admonition-title:before {
+      @svg: escape(
+        '<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="@{yellow}" viewBox="0 0 16 16"><path d="M8.982 1.566a1.13 1.13 0 0 0-1.96 0L.165 13.233c-.457.778.091 1.767.98 1.767h13.713c.889 0 1.438-.99.98-1.767L8.982 1.566zM8 5c.535 0 .954.462.9.995l-.35 3.507a.552.552 0 0 1-1.1 0L7.1 5.995A.905.905 0 0 1 8 5zm.002 6a1 1 0 1 1 0 2 1 1 0 0 1 0-2z"/></svg>'
+      );
+      content: url("data:image/svg+xml,@{svg}");
+    }
+    .admonition.danger {
+      color: @red;
+      background-color: fade(@red, 25%);
+    }
+    .admonition.danger > .admonition-title:before,
+    .admonition.error > .admonition-title:before {
+      @svg: escape(
+        '<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="@{red}" viewBox="0 0 16 16"><path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 4a.905.905 0 0 0-.9.995l.35 3.507a.552.552 0 0 0 1.1 0l.35-3.507A.905.905 0 0 0 8 4zm.002 6a1 1 0 1 0 0 2 1 1 0 0 0 0-2z"/></svg>'
+      );
+      content: url("data:image/svg+xml,@{svg}");
+    }
```

Comment:
Some nesting would be nice here.

Reply:
this one should be solved with [da5f71b](https://github.com/catppuccin/userstyles/pull/1198/commits/da5f71bf52333406bef35ae7cf3780e2a9ad324a)